packages/react-core/src/components/NumberInput/NumberInput.tsx

```diff
@@ -83,7 +83,7 @@ export const NumberInput: React.FunctionComponent<NumberInputProps> = ({
           variant="control"
```

Comment:
Can you modify lines 52 and 54 to read like `onMinus = () => {},` and `onPlus = () => {}` instead? This is more consistent with our style across other components and generates nicer code.